lualib/modular_admin/modular_admin_spectate.lua

```diff
@@ -156,7 +156,7 @@ function modular_admin_spectate_set_normal_teleport(p)
 	modular_admin_spectate_set_normal(p)
 	if global.modular_admin_spectate.player_spectator_state[p.index] == false then
 		p.print("Teleporting you to the location you are currently looking at.")
-		p.teleport(p.surface.find_non_colliding_position("player", pos))
+		p.teleport(p.surface.find_non_colliding_position("player", pos, 0, 1))
```

Comment:
caused crash.. 
`190267.216 Error MainLoop.cpp:1035: Exception at tick 7693249: Error while running event level::on_tick (ID 0)
...factorio/server2/temp/currently-playing/lualib/event.lua:294: ...-playing/lualib/modular_admin/modular_admin_spectate.lua:111: bad argument #0 to 'find_non_colliding_position' (Wrong number of arguments.)`


`190267.216 Error ServerMultiplayerManager.cpp:96: MultiplayerManager failed: "Error while running event level::on_tick (ID 0)
...factorio/server2/temp/currently-playing/lualib/event.lua:294: ...-playing/lualib/modular_admin/modular_admin_spectate.lua:111: bad argument #0 to 'find_non_colliding_position' (Wrong number of arguments.)"`

Reply:
Fixed with latest commit. Also you highlighted the wrong faulting line. Line 111 had the bad argument in the stack trace, you highlighted line 159.